ops/testing.py

```diff
@@ -1075,6 +1075,8 @@ def __init__(self, unit_name, meta):
         self.app_name = self.unit_name.split('/')[0]
         self.model_name = None
         self.model_uuid = 'f2c1b2a6-e006-11eb-ba80-0242ac130004'
+
+        self._harness_tmp_dir = tempfile.TemporaryDirectory(prefix='ops-harness-')
```

Comment:
Does this require an explicit `cleanup()`? To be added to the harness cleanup method?

Reply:
No - it's auto cleaned up when the object dies - or when the context ends if it's used inside one.